src/Microsoft.VisualStudio.Services.Agent/ThrottlingReportHandler.cs

```diff
@@ -39,6 +40,10 @@ public ThrottlingReportHandler(IThrottlingReporter throttlingReporter)
 
         protected async override Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
         {
+#if OS_LINUX
+            request.Version = HttpVersion.Version11;
```

Comment:
i don't think we need this, this is still using the vsshttpmessagehandler

Reply:
Yeah, let me double check. I introduced this code back from when I took the original approach of specifying the version when getting the HTTP client.

Reply:
This override is not needed. Removed.